application/src/main/resources/ecc.yml

```diff
@@ -53,10 +53,20 @@ connection:
     ## Allow routing requests directly to a remote datacenter.
     ## This allows locks for other datacenters to be taken in that datacenter instead of via the local datacenter.
     ## If clients are prevented from connecting directly to Cassandra nodes in other sites this is not possible.
-    ## If remote routing is disabled its not possible to use LOCAL_SERIAL consistency for the locking,
-    ## instead SERIAL consistency will be used for those request.
+    ## If remote routing is disabled, instead SERIAL consistency will be used for those request.
     ##
     remoteRouting: true
+    ##
+    # # Allow to override consistency level for LWT (lightweight transactions). Possible values are:
```

Comment:
Is there a reason for having spaces between "#"? If not then remove them.

Reply:
Done